pkg/options/config.go

```diff
@@ -0,0 +1,36 @@
+package options
+
+import (
+	"embed"
+	"fmt"
+
+	"github.com/BurntSushi/toml"
+	"github.com/lilypad-tech/lilypad/pkg/data"
+	"github.com/lilypad-tech/lilypad/pkg/web3"
+)
+
+//go:embed configs
+var fs embed.FS
+
+type Config struct {
+	Web3          web3.Web3Options   `toml:"web3"`
+	ServiceConfig data.ServiceConfig `toml:"services"`
+}
+
+// TODO(bgins) Check for user-defined config files
+func getConfig(network string) (*Config, error) {
+	var config Config
+
+	path := fmt.Sprintf("configs/%s.toml", network)
```

Comment:
Curious as to why you went with `fmt.Sprintf` to concatenate, it seems to be the slowest method ([ref](https://dev.to/schadokar/the-fastest-way-to-concatenate-strings-in-golang-5mf))

Reply:
In a word, naivety. 😁 

But reading up on it a bit, I think it's an acceptable choice for this use case:

- We aren't in a tight loop or situation where we are constructing strings that often. We'll do this once per command run.
- It's more readable that using the `+` operator. We know immediately what we are doing, without considering operands to `+`.
- `Strings.builder` would take multiple lines of code and would be less readable

Open to the other options, but for code readability this seems like the best option. Thoughts?

Reply:
Your reasoning works for me